setup.py

```diff
@@ -84,7 +84,7 @@
 c_files = []
 for root, dirs, files in os.walk(PACKAGENAME):
     for filename in files:
-        if filename.endswith('.c'):
+        if filename.endswith('.c') or filename.endswith('.pxd'):
```

Comment:
Minor point, but should we rename the `c_files` variable while we are at it (since it's no longer accurate)?  Maybe "compiled_files"?